benchmarks/continous/db/tool/orm/src/models.rs

```diff
@@ -6,10 +6,16 @@ use crate::schema::ft_transfers;
 
 #[derive(Insertable, Deserialize)]
 #[diesel(table_name = ft_transfers)]
+#[diesel(check_for_backend(diesel::pg::Pg))]
 pub struct NewFtTransfer {
-    // TODO store start and time in two separate columns
-    /// String representation of UTC datetime when the benchmark was run, e.g. '2024-06-07T11:30:44Z'
+    /// String representation of UTC datetime when the benchmark run was started, e.g.
+    /// '2024-06-07T11:30:44Z'.The name `time` (as opposed to `time_start`) is chosen to enable
+    /// Grafana [time series queries].
+    ///
+    /// [time series queries]: https://grafana.com/docs/grafana/latest/datasources/postgres/#time-series-queries
     pub time: DateTime<Utc>,
```

Comment:
Do we want `time_start` or `time_begin` here maybe?

Reply:
Having a timestamp field named only `time` allows Grafana to pick it up as a time series, see the comment in line 12 of this file.

Reply:
Oh, sorry, thanks!